bitnami/redis/README.md

```diff
@@ -180,6 +180,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `master.persistence.annotations`            | Additional custom annotations for the PVC                                                         | `{}`            |
 | `master.persistence.selector`               | Additional labels to match for the PVC                                                            | `{}`            |
 | `master.persistence.existingClaim`          | Use a existing PVC which must be created manually before bound                                    | `""`            |
+| `master.persistence.dataSource`             | Custom PVC data source                                                                            | `{}`            |
```

Comment:
Please note that you need to add these new values to the **values.yaml** and update the README.md metadata accordingly using the [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool.